src/all/luscious/src/eu/kanade/tachiyomi/extension/all/luscious/Luscious.kt

```diff
@@ -55,6 +56,11 @@ abstract class Luscious(
 
     private val json: Json by injectLazy()
 
+    override fun headersBuilder(): Headers.Builder {
+        return Headers.Builder()
+            .add("Referer", getMirrorPref() ?: "https://www.luscious.net/")
+    }
```

Comment:
```suggestion
    override fun headersBuilder(): Headers.Builder {
        return super.headersBuilder()
            .add("Referer", "$baseUrl/")
    }
```
preserve app provided useragent

Reply:
Thank you for the review !